vj4/model/adaptor/training.py

```diff
@@ -8,16 +8,17 @@
 
 
 @argmethod.wrap
-async def add(domain_id: str, title: str, content: str, owner_uid: int, dag=[]):
+async def add(domain_id: str, title: str, intro: str, content: str, owner_uid: int, dag=[]):
```

Comment:
Just change the name of field `content` to introduction, because field `content` already has been shown in training list. Add `description` field if you like.